apps/server/src/modules/files-storage/client/s3-client.adapter.ts

```diff
@@ -28,17 +29,10 @@ export class S3ClientAdapter implements IStorageClient {
 
 	// is public but only used internally
 	public async createBucket() {
-		try {
-			this.logger.log({ action: 'create bucket', params: { bucket: this.config.bucket } });
+		this.logger.log({ action: 'create bucket', params: { bucket: this.config.bucket } });
```

Comment:
We can not pass errors from external systems without re-throwing and let them into our same domain area.
The AC from ticket for this situation is 
throw new InternalServerErrorException(......, 'S3ClientAdapter:createBucket', {cause: err}); 

The ticket has a very small scope. The change you did in this file and all similar places are not part of the ticket.